src/style.css

```diff
@@ -1,15 +1,15 @@
 
 .recent-scores {
   display: flex;
-   align-items: center; 
-   gap: 3rem;
-   margin-left: 10rem;
+  align-items: center;
+  gap: 3rem;
+  margin-left: 10rem;
```

Comment:
- [ ] Kindly make your projects work correctly with Webpack. Modify your `.js` file to match the name that you setup in your configuration. Doing this the error should disappear when building the app and will load styles correctly

![image](https://github.com/Young-Gaius/Leaderboard/assets/93630700/13cf31ad-0cc2-42f8-8817-c91fb1aa9f50)